<ui version="4.0" >
 <class>ProfileWidgetUI</class>
 <widget class="QWidget" name="ProfileWidgetUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>430</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QListView" name="listView" />
      </item>
      <item row="0" column="1" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="KPushButton" name="kpushbutton_add" >
          <property name="text" >
           <string>Add...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="kpushbutton_rem" >
          <property name="text" >
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="kpushbutton_mod" >
          <property name="text" >
           <string>Modify...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="kpushbutton_copy" >
          <property name="text" >
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="kpushbutton_wizard" >
          <property name="text" >
           <string>Assistant...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>93</width>
            <height>131</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KPushButton" name="kpushbutton_load" >
          <property name="text" >
           <string>Load...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="kpushbutton_save" >
          <property name="text" >
           <string>Save...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
